wcfsetup/setup/db/install.sql

```diff
@@ -1446,6 +1446,20 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY (urlHash),
+	title VARCHAR(255) NOT NULL DEFAULT '',
+	description TEXT,
+	imageUrl TEXT NOT NULL DEFAULT '',
```

Comment:
This can't have a real default value (Error in query (1101): BLOB, TEXT, GEOMETRY or JSON column 'imageUrl' can't have a default value ). I suggest:
```suggestion
	imageUrl TEXT DEFAULT NULL,
```